config/locales/es.yml

```diff
@@ -92,10 +93,10 @@ es:
         header: Conoce a los Administradores
         about: Acerca de Nosotros
         news: "%{name} Noticias"
-        photo-alt-text: Foto de %{name}
+        photo-alt-text: "Foto de %{name}"
       apply-for-grant: Solicita una beca
       inactive-chapter: Este capítulo está inactivo.
-      inactive-chapter-reason: Un capítulo inactivo es uno que no ha otorgado una beca en seis meses. Si estás interesado en reiniciar este capítulo, por favor <a href="mailto:join@awesomefoundation.org">comunícate con nosotros</a>.
+      inactive-chapter-reason: Un capítulos inactivo es uno que no ha otorgado una beca en seis meses. Si estás interesado en reiniciar este capítulo, por favor <a href="mailto:join@awesomefoundation.org">comunícate con nosotros</a>.
```

Comment:
Just confirming - is this correct as it is referring to a single chapter, not multiples.

Reply:
I had that incorrect, sorry. I changed it to reflect capitulo